### PR TITLE
Updated Figure documentation.

### PR DESCRIPTION
**Pull request by mattiasberlamont**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the styleguide CHANGELOG accordingly.

### CODE REVIEW

File: components/31-molecules/figure/README.md [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
 
 ## When to use this component
 
-Use the figure component to present an image on a page.
+Use the image component to present an image on a page.
 
 ## How it works
 
-The figure component wraps an image in a container and adds the benefit of providing an optional caption tag.
+The image component wraps an image in a container and adds the benefit of providing an optional caption tag.
```
Comment (delrueba) [inline components/31-molecules/figure/README.md]:
```suggestion
The image component wraps an image in a figure and adds the benefit of providing an optional caption tag.
```

**Review by delrueba — CHANGES_REQUESTED** [review]
Can you also add that the alt attribute is required and must be empty for purely decorative images.

There's a lot of content online about writing a good alt-text, but that might lead us too far, 
perhaps it's sufficient to reference the official WCAG Technique page: https://www.w3.org/TR/WCAG20-TECHS/H37.html

**Review by hstandaert — APPROVED** [review]
Might be useful to add some information about the fact that a ratio should always be provided, that we use `8:5` as a default and a placeholder image is loading or not found.

**Comment by mattiasberlamont** [comment]
Should purely decorative images be wrapped in a figure?

**Comment by mattiasberlamont** [comment]
@hstandaert @delrueba. After discussing this with @delrueba I think it would be better to keep the name "Figure" for this component. I updated the documentation and also added the requirements for images you mentioned. The references from the Image Gallery en Image Wall component documentation were also updated. Could you please review the changes?

**Review by delrueba — APPROVED** [review]
OK for me

**Comment by mattiasberlamont** [comment]
@hstandaert OK for you?

**Review by hstandaert — APPROVED** [review]

**Pull request merged**